### DISCUSSION TITLE
Interfaces between a gap block and two blocks

### DISCUSSION THREAD

**Discussion by styyokuda** [idaholab/moose#28237] [Q&A Meshing]
All,

I am trying to generate interfaces between blocks and gap blocks to apply InterfaceKernels objets.
Please see attached "Interfaces between 2 blocks.pdf" for the procedure I am using.
Also, Attached "gapBTW2blocks_mesh.pdf" is the input script.
It is not working. Please help.  S. Thomas

[Interfaces between 2 blocks.pdf](https://github.com/user-attachments/files/16384390/Interfaces.between.2.blocks.pdf)
[gapBTW2blocks_mesh.pdf](https://github.com/user-attachments/files/16384391/gapBTW2blocks_mesh.pdf)

**Comment by GiudGiud** [comment]
Hello

I m curious why you feel the need to introduce a gap block between domain 1 and 2. 
Why cannot you use interface kernels directly at the interface between 1 and 2?

Guillaume
